Description: Ignore some doctests
 We do not package fonts/font.ttf, hence these doctests fail to compile
Author: NoisyCoil <noisycoil@tutanota.com>
Forwarded: not-needed
Last-Update: 2025-04-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -4,7 +4,7 @@
 //! Re-exports `ttf_parser::*`.
 //!
 //! # Example
-//! ```
+//! ```ignore
 //! use owned_ttf_parser::{AsFaceRef, Face, OwnedFace};
 //!
 //! # let owned_font_data = include_bytes!("../fonts/font.ttf").to_vec();
--- a/src/owned.rs
+++ b/src/owned.rs
@@ -12,7 +12,7 @@
     /// You can set index for font collections. For simple ttf fonts set index to 0.
     ///
     /// # Example
-    /// ```
+    /// ```ignore
     /// # use owned_ttf_parser::OwnedFace;
     /// # let owned_font_data = include_bytes!("../fonts/font.ttf").to_vec();
     /// let owned_face = OwnedFace::from_vec(owned_font_data, 0).unwrap();
@@ -40,7 +40,7 @@
     /// Extracts a slice containing the data passed into [`OwnedFace::from_vec`].
     ///
     /// # Example
-    /// ```
+    /// ```ignore
     /// # use owned_ttf_parser::OwnedFace;
     /// # let owned_font_data = include_bytes!("../fonts/font.ttf").to_vec();
     /// let data_clone = owned_font_data.clone();
@@ -54,7 +54,7 @@
     /// Unwraps the data passed into [`OwnedFace::from_vec`].
     ///
     /// # Example
-    /// ```
+    /// ```ignore
     /// # use owned_ttf_parser::OwnedFace;
     /// # let owned_font_data = include_bytes!("../fonts/font.ttf").to_vec();
     /// let data_clone = owned_font_data.clone();
--- a/src/preparse.rs
+++ b/src/preparse.rs
@@ -12,7 +12,7 @@
 /// `.as_face_ref()` equivalents that must parse their subtables on each call.
 ///
 /// # Example
-/// ```
+/// ```ignore
 /// use owned_ttf_parser::{AsFaceRef, GlyphId, OwnedFace, PreParsedSubtables};
 ///
 /// # let owned_font_data = include_bytes!("../fonts/font.ttf").to_vec();
